packer/scripts/riot.sh

```diff
@@ -16,6 +16,10 @@ cd /opt && wget -nv -O - "${ARM_GCC_ARCHIVE_URL}" | tar -jxf -
 
 echo "export PATH=/opt/gcc-arm-none-eabi-7-2018-q2-update/bin:\$PATH" >> /home/user/.bashrc
 
+# Add .local/bin to PATH
+echo "export PATH=$PATH:~/.local/bin" >> /etc/profile
```

Comment:
You have an example just above. Please use the same strategy, e.g:
```suggestion
echo "export PATH=\$PATH:home/user/.local/bin" >> /home/user/.bashrc
```

Reply:
It was to source user instaled libs, should I install everything as sudo?

Reply:
Or let user handles this.

Reply:
> user scope libs

what libs are talking about ? what is "everything" ?

Reply:
Sorry I wrote this a bit in a rush. I usually add `~/.local/bin` to `PATH` to have for when installing using `--user` and if no `home/bin`. Since you might get conflicts with `pip` libs, I usually install using `--user` or in a `virtualenv`. I though installing as `user` could be better?